app/src/src/screens/tabs/SavedScreen.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import SavedNavigator from '../../navigators/SavedNavigator';
+
+const SavedScreen = ({ navigation }) => {
+  return <SavedNavigator />;
+};
+
+export default SavedScreen;
```

Comment:
This file shouldn't be under the tabs folder because it is not a tab. We probably don't need this file at all since I don't see where it's being used. We can probably delete this.

Reply:
deleted it